decidim-admin/app/views/decidim/admin/moderations/reports/index.html.erb

```diff
@@ -1,105 +1,105 @@
+                                t("actions.expand", scope: "decidim.moderations"),
+                                class: "action-icon--expand" %>
+            </td>
           </tr>
-        </thead>
-        <tbody>
-          <% reports.each do |report| %>
-            <tr data-id="<%= report.id %>">
-              <td><%= t("decidim.admin.moderations.report.reasons.#{report.reason}") %></td>
-              <td><%= report.locale.present? ? locale_name(report.locale) : "" %></td>
-              <td><%= report.details&.truncate(250) %></td>
-              <td class="actions">
-                <%= icon_link_to "fullscreen-line",
-                                  moderation_report_path(moderation_id: moderation, id: report.id),
-                                  t("actions.expand", scope: "decidim.moderations"),
-                                  class: "action-icon--expand" %>
-              </td>
-            </tr>
-          <% end %>
-        </tbody>
-      </table>
-    </div>
-    <div class="form__wrapper-block flex-col-reverse md:flex-row justify-between">
-      <% if !moderation.reportable.hidden? && allowed_to?(:unreport, permission_resource) %>
-        <%= link_to t("actions.unreport", scope: "decidim.moderations"), unreport_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary" %>
-      <% end %>
-      <% if !moderation.reportable.hidden? && allowed_to?(:hide, permission_resource) %>
-        <%= link_to t("actions.hide", scope: "decidim.moderations"), hide_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary alert" %>
-      <% end %>
-      <% if moderation.reportable.hidden? && allowed_to?(:unhide, permission_resource) %>
-        <%= link_to t("actions.unhide", scope: "decidim.moderations"), unhide_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary alert" %>
-      <% end %>
-    </div>
+        <% end %>
+      </tbody>
+    </table>
+  </div>
+  <div class="form__wrapper-block flex-col-reverse md:flex-row justify-between">
+    <% if !moderation.reportable.hidden? && allowed_to?(:unreport, permission_resource) %>
+      <%= link_to t("actions.unreport", scope: "decidim.moderations"), unreport_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary" %>
+    <% end %>
+    <% if !moderation.reportable.hidden? && allowed_to?(:hide, permission_resource) %>
+      <%= link_to t("actions.hide", scope: "decidim.moderations"), hide_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary alert" %>
+    <% end %>
+    <% if moderation.reportable.hidden? && allowed_to?(:unhide, permission_resource) %>
+      <%= link_to t("actions.unhide", scope: "decidim.moderations"), unhide_moderation_path(id: moderation), method: :put, class: "button button__sm button__secondary alert" %>
+    <% end %>
```

Comment:
I think these two buttons should be stickys as they're the primary and more important action